tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Sql/SqlVisitor.cs

```diff
@@ -207,13 +207,25 @@ protected static string GetRowStructNameForTableType(SchemaObjectName objectName
         }
 
         /// <summary>
-        /// Strips away the version suffix from a table type name.
+        /// Strips away the underscore in a in a suffix for a table type name.
+        /// For example, MyTableType_1 will return MyTableType1
         /// </summary>
         /// <param name="objectName">The table type name</param>
         /// <returns>The name</returns>
         private static string GetTableTypeNameWithoutVersionSuffix(SchemaObjectName objectName)
         {
-            return Regex.Replace(objectName.BaseIdentifier.Value, @"_\d+", string.Empty);
+            return Regex.Replace(objectName.BaseIdentifier.Value, @"_(\d+)$", "$1");
```

Comment:
I can't remember if we wanted to have the V in there too, do you remember? `MyTableTypeV1`

Reply:
That's a good idea, I like it!

Reply:
Fixed [here](https://github.com/microsoft/healthcare-shared-components/pull/130/commits/5fe612c6c12751cb1b1f0284e9e589adb86e950a)